### PR TITLE
Gui

### PR DESCRIPTION
**Pull request by PurnaChandraPanda**

### CODE REVIEW

File: src/CollectSFDataGui/Server/Startup.cs [hunk 1]
```diff
@@ -22,12 +24,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 app.UseDeveloperExceptionPage();
                 app.UseWebAssemblyDebugging();
+
+                app.UseSwagger();
+                app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "WebApplication1 v1"));
```
Comment (jagilber) [inline src/CollectSFDataGui/Server/Startup.cs]: can you give swagger endpoint a real name like CollecsfdataGuiEndpoint or something?

File: src/CollectSFDataGui/Server/Startup.cs [hunk 2]
```diff
@@ -47,6 +53,12 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllersWithViews();
             services.AddRazorPages();
+            services.AddSingleton<IDataService, DataService>();
+
+            services.AddSwaggerGen(c =>
+            {
+                c.SwaggerDoc("v1", new OpenApiInfo { Title = "WebApplication1", Version = "v1" });
```
Comment (jagilber) [inline src/CollectSFDataGui/Server/Startup.cs]: can you give swagger endpoint a real name like CollecsfdataGuiEndpoint or something?

**Review by jagilber — COMMENTED** [review]
hey @PurnaChandraPanda can you set the swagger endpoint to a real name?

File: src/CollectSFDataGui/Server/Startup.cs (hunk 1 above)
Comment (PurnaChandraPanda) [reply src/CollectSFDataGui/Server/Startup.cs]: suggestion is added

File: src/CollectSFDataGui/Server/Startup.cs (hunk 2 above)
Comment (PurnaChandraPanda) [reply src/CollectSFDataGui/Server/Startup.cs]: suggestion is added in new commit

**Pull request merged**